R/parser.R

```diff
@@ -4,6 +4,281 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+#' Parse Bugzilla data obtained from Perceval traditional Bugzilla backend
+#'
+#' @param perceval_path path to perceval binary
+#' @param bugzilla_json json object containing bugzilla issue data
+#' @param comments if true, the comments are parsed along with the issues
+#' @export
+#' @family parsers
+parse_bugzilla <- function(bugzilla_json,comments=FALSE){
+
+  json_issue_comments <- data.table(jsonlite::stream_in(textConnection(bugzilla_json),verbose = FALSE))
```

Comment:
I will retract the comment above. It seems at least for the retro API of bugzilla via perceval, the file is downloaded as a list of json files, where each line is a different json file. This will cause the `parse_json` function to fail (contrary to your Crawler, which worked for me). 

This being the case, I guess we are without much option than representing the list as a table of lists (which I find confusing to manipulate, as simply a list of lists).